wgpu-hal/src/dx12/shader_compilation.rs

```diff
@@ -176,6 +178,12 @@ mod dxc {
             &[],
         );
 
+        let format_error_message = |e: DxcOperationResult| {
```

Comment:
I generally prefer to make stuff like this a function, but it isn't used anywhere else and it's short enough that it isn't a big deal.

Reply:
meh, I'd rather not infect the project with a change of style if possible. How about a private function in the dxc module, like `get_error_string_from_dxc_result`?

Reply:
We generally stay away from lambdas, yeah. A private function would be fine.